compiler/tests-ocaml/lib-effects/dune

```diff
@@ -47,3 +54,18 @@
     (run node %{test}))
    (run cat)))
  (modes js wasm))
+
+(tests
```

Comment:
I  try to keep compiler/tests-ocaml for tests coming from the ocaml repo. Could you move it to `compiler/tests-jsoo/lib-effecfs/` ?

(I need to add a readme here to explain the layout.)